lib/SIL/SILBasicBlock.cpp

```diff
@@ -355,3 +355,7 @@ bool SILBasicBlock::isTrampoline() const {
     return false;
   return begin() == Branch->getIterator();
 }
+
+bool SILBasicBlock::isLegalToHoistInto() const {
+  return true;
```

Comment:
Yes. I talked with Arnold and I don't think we use this. If you do extract this, can you change the return to false so we are conservative?